app/views/teams/_form.html.slim

```diff
@@ -39,7 +39,6 @@
     = f.link_to_add 'Add another member', :roles, class: 'btn btn-primary col-sm-offset-3'
 
   - if f.object.accepted?
-    = f.input :project_name, placeholder: 'Name of your project', label: 'Project Name'
```

Comment:
If you like, you can still show the project name here, with something like: `f.object.project`
(So that the user doesn't need to go and find the correct Project Name, when they are telling something about the project in the `description field`)

Reply:
Yes, thought about this already.

Reply:
The project doesn't need to be submitted, right? `readonly` does submit, `disabled` doesn't. Also, `disabled` will make sure the field is skipped when navigating with tabbing. Looks like a more appropriate choice here. What do you think?